ci/README.md

```diff
@@ -53,7 +53,17 @@ enterprise-master, the check-merge-to-enterprise will fail. Say your branch name
 is `$PR_BRANCH`, we will refer to `$PR_BRANCH` on community as
 `community/$PR_BRANCH` and on enterprise as `enterprise/$PR_BRANCH`. If the
 job already passes, you are done, nothing further required! Otherwise follow the
-below steps. First make sure these two things are the case:
+below steps.
```

Comment:
Does it make sense to explain when the job will pass so that people have an idea of what this job is checking?

- The job will fail if enterprise/$PR_BRANCH is not ancestor of community/$PR_BRANCH
- The job will fail if merging community/$PR_BRANCH to enterprise-master creates conflicts.